### DISCUSSION TITLE
Suricata Disappeared: No such container

### DISCUSSION THREAD

**Discussion by pappajohndd** [Security-Onion-Solutions/securityonion#2791] [Unsupported Versions] [answered]
Distributed deployment centos 7 latest SO version.
Getting no Suricata alerts, tried to restart Suricata using sudo salt \*_sensor cmd.run 'so-suricata-restart'
    =========================================================================
    Restarting suricata...

    This could take a while if another Salt job is running.
    Run this command with --force to stop all Salt jobs before proceeding.
    =========================================================================
    Error response from daemon: No such container: so-suricata
    Error: No such container: so-suricata
    [ERROR   ] Rendering exception occurred
    Traceback (most recent call last):

**Comment by weslambert** [comment]
Are you able to find any clues by running something like `docker logs so-suricata`?

**Comment by J0hnnyb0y86** [comment]
have you solved? i have the same problem

**Comment by pappajohndd** [comment, marked as answer]
Never solved it. I completely recreated the entire deployment.

John Freeman | Network Security Engineer
220 S. First Street, Broken Arrow, OK 74012
Office: 918.259.2400 ext 5357
City of Broken Arrow
This message is intended only for the use of the individual or entity to which it is
addressed and may contain information that is privileged, confidential and exempt
from disclosure under applicable law. If you receive this message in error, you are
hereby notified that the City of Broken Arrow does not consent to any reading,
dissemination, distribution or copying of this message.  If you have received this
communication in error, please notify the sender immediately and destroy the
transmitted information immediately.

**Reply by IzacFrank** [reply]
In my case, Elastalert just disappeared. No such container #6966 and I couldn't find any solution, either.

**Comment by chrisshawio** [comment]
In my case, this error occurred after editing the global.sls file with incorrect syntax. After correcting the syntax everything worked fine.

**Reply by sanba06c** [reply]
Can you show what the specific error in the syntax is?

**Reply by dougburks** [reply]
@sanba06c If you have questions or problems, please start a new discussion rather than replying to an old discussion.

**Comment by sanba06c** [comment]
The following command line was executed successfully:

salt \* cmd.run 'so-suricata-restart'
